package.json

```diff
@@ -66,6 +65,7 @@
     "ember-cli-deploy-git-ci": "^1.0.1",
     "ember-cli-eslint": "^5.1.0",
     "ember-cli-inject-live-reload": "^2.0.1",
+    "ember-cli-sass": "^10.0.0",
```

Comment:
I think this should still be a dependency, why make it a dev dependency?

Reply:
my mistake